### DISCUSSION TITLE
Scrolling is not enabled in the test summary section using the desktop app

### DISCUSSION THREAD

**Discussion by rodrigomf24** [postmanlabs/postman-flows#294] [Defect]
After running the flow a few times, I realized I can't scroll to see the latest test summaries in the desktop app

<img width="512" alt="Screen Shot 2022-04-08 at 8 54 02 AM" src="https://user-images.githubusercontent.com/3925338/162478476-c6f48795-d1e3-41c1-a587-618b6ae9919e.png">

**Comment by dev-kanishk** [comment]
Hey, @rodrigomf24 thank you for informing us.  Fixing the issue right now.  in the next release issue will be resolved.
